README.md

```diff
@@ -50,15 +50,6 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
 
 ## Preview Features
 
-* View and edit a web app's files
-  * To enable this feature, click File > Preferences > Settings. modify your `appService.showRemoteFiles` to be true.
-
-  ![Enable Remote File Editing](resources/Remote_File_Editing_Setting.png)
-
-  * To view a file, click on it in the explorer.
```

Comment:
I understand removing this from the 'Preview Features' section, but do you want to move some of the bullets here to the 'Features' section?